Specs/Scene/CameraSpec.js

```diff
@@ -993,6 +993,53 @@ defineSuite([
         expect(camera.right).toEqual(right);
     });
 
+    it('moves forward in 2D mode zooms in camera instead of moving it', function() {
+        var frustum = new OrthographicOffCenterFrustum();
+        frustum.near = 1.0;
+        frustum.far = 2.0;
+        frustum.left = -2.0;
+        frustum.right = 2.0;
+        frustum.top = 1.0;
+        frustum.bottom = -1.0;
+        camera.frustum = frustum;
```

Comment:
Do you need to set the frustum for this test? I don't see these values used, only that the camera's frustum changes.

Reply:
Same with all the tests below.

Reply:
I just checked - they do need it because an OrthographicOffCenterFrustum is required for 2D mode.